lib/net_suite/client.rb

```diff
@@ -44,6 +52,17 @@ def #{method}(url = nil, body = nil, headers = nil, &block)
 
     attr_accessor :retry_count
 
+    def trace_call(method, url, request_payload, *_args, &)
```

Comment:
`*_args` isn't used, so let's omit it

```suggestion
    def trace_call(method, url, request_payload, &)
```